spec/controllers/api/v1/chapters_controller_spec.rb

```diff
@@ -1,193 +1,229 @@
-RSpec.describe Api::V1::ChaptersController, 'GET #show' do
-  render_views
+RSpec.describe Api::V1::ChaptersController do
```

Comment:
Promoting this to a request spec was one step too far

Reply:
Nah, better to minimise changes anyway :+1: